### PR TITLE
import connect.sh

### PR DESCRIPTION
**Pull request by goneri**
connect.sh is a little helper script you can use to connect to your
install-server.

### CODE REVIEW

File: common/infra-virt.function [hunk 1]
```diff
@@ -217,6 +217,26 @@ find_host_by_profile() {
...
+        if [[ $vm_prefix != $PREFIX ]]; then
+            continue
+        fi
+
+        local vm_profile=$(get_key_from_domain_metadata ${name} profile)
+        if [[ $vm_profile == "install-server" ]]; then
+            continue
+        fi
+
+        ssh ${SSHOPTS} root@${virthost} virsh $name
```
Comment (cedlecomte) [inline common/infra-virt.function]: Maybe `virsh reboot` will be better :-)

File: connect.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+for f in $(sudo find /etc/libvirt/qemu -name "${prefix}_*"); do
+    role=$(sudo xmllint --xpath "/domain/metadata/*[local-name()='instance']/*[local-name()='profile']/text()" ${f})
+    if [ "${role}" == "install-server" ]; then
+        echo "domain file found at ${f}"
+        domain_file=${f}
+    fi
+done
+
+mac=$(sudo xmllint --xpath 'string(/domain/devices/interface[last()]/mac/@address)' ${domain_file})
+ip=$(awk "/${mac}/ {print \$3}" /var/lib/libvirt/dnsmasq/nat.leases)
```
Comment (cedlecomte) [inline connect.sh]: Maybe you can use get_ip and get_mac with a sourcing of infra-virt.common

**Comment by goneri** [comment]
That's a good idea. I will do that.

**Comment by goneri** [comment]
@Prophidys done. Indeed, the script way more shorter but also slower. It seems the virsh metadata call is not that fast.

**Pull request merged**